src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/arrivals/ArrivalsService.kt

```diff
@@ -57,34 +59,46 @@ class ArrivalsService(
 
     val arrival = getArrival(moveId)
 
+    return if (!arrival.isCurrentPrisoner)
+      when (arrival.prisonNumber) {
+        null -> createAndAdmitOffender(confirmArrivalDetail, moveId)
+        else -> admitOffender(confirmArrivalDetail, moveId, arrival.prisonNumber)
+      } else
+      throw IllegalArgumentException("Feature not available.")
```

Comment:
can we make this more descriptive: `Do not support confirming arrival of active prisoner: ${arrival.prisonNumber}`

Reply:
or summat

Reply:
done